projects/plugins/boost/app/assets/src/js/main.tsx

```diff
@@ -41,6 +42,20 @@ const useBoostRouter = () => {
 				</SettingsPage>
 			),
 		},
+		{
+			path: '/cache-debug-log',
+			loader: () => {
+				if ( shouldGetStarted ) {
+					return redirect( '/getting-started' );
+				}
+				return null;
```

Comment:
Not for fixing in this PR, but we repeat the shouldGetStarted check over and over in main.tsx?!

Reply:
I agree, I have addressed it.